HomeUI/src/views/apps/RegisterFluxApp.vue

```diff
@@ -1197,8 +1359,8 @@ export default {
       deep: true,
     },
   },
-  beforeMount() {
-    this.appRegistrationSpecification = this.appRegistrationSpecificationv4template;
+  beforeMount() { // TODO - revert to v4
```

Comment:
Should this TODO be completed? Or is it okay to leave?

Reply:
This needs to be reverted but to test needs to be this way.